### DISCUSSION TITLE
Welcome to mfsbsd-13.1-script Discussions!

### DISCUSSION THREAD

**Discussion by bretton** [depenguin-me/depenguin-run#36] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to have discussions that aren't suited to the issues page.
  * Ask questions you’re wondering about.
  * Share ideas.

To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by click0** [comment]
From rescue Linux hosters based on Debian packages? 
or have you seen the use of other package bases (yum/pacman/..)?

**Reply by bretton** [reply]
[test request Equinix Metal](https://github.com/depenguin-me/mfsbsd-13.1-script/issues/46) lists a recovery console which uses Alpine linux. 

[test request Servers.com Dell Servers](https://github.com/depenguin-me/mfsbsd-13.1-script/issues/43) lists "DMGMTSrvInit 1.0 x86_64 [based on CentOS 6 x86_64]"

[test request Scaleway Dedibox](https://github.com/depenguin-me/mfsbsd-13.1-script/issues/44) lists Ubuntu, but they also have a FreeBSD recovery console

**Comment by click0** [comment]
You also need a list of Linux live isos (preferably on different package bases), which start the ssh service at startup.

**Reply by bretton** [reply]
Is there a shortcut to this? most -server versions of ISOs have it, but the desktop live CDs don't. 

google isn't being that helpful, there is also an absence of recent info, much of it is 5+ years old.

[vultr](https://www.vultr.com/docs/troubleshoot-your-vps-with-bootable-isos/) lists:
* Finnix
* SystemRescue x64

To be tested:
* Arch
* Alpine Linux
* Debian (likely)
* DMGMTSrvInit 1.0 x86_64 (based on CentOS 6 x86_64) if available
* Ubuntu-server (likely)

For my own testing purposes I'll write ISO to USB drive, boot from it with a second wiped USB drive attached too, then install the version of Linux to the second USB drive. 
Then just boot from the second USB drive when needing that distro.

**Reply by click0** [reply]
Between the actions "boot from USB" and "connect via ssh". 
You need to do some manual steps:

- create root account
- install ssh daemon
- allow ssh connection as root
- start sshd daemon
- (optional) add allow rules to firewall

All these manual actions interfere with automated testing.

**Reply by click0** [reply]
So far I see options:
- [MfsLinux](https://mfsbsd.vx.sk/files/iso/mfslinux/mfslinux-0.1.10-f9c75a4.iso) (based Openwrt) - `root`/`mfsroot`
- [Archlinux-zfs-nox](https://cloud.s-morozov.net/s/472pNPRdZ6RGiT8) (based ArchLinux) - `root`/`root`. [Instructions for building](https://git.s-morozov.net/sergey/archlive-zfs) your own image.
